### PR TITLE
Update name of RHCOS

### PR DESCRIPTION
**Pull request by ashcrow**
Update `troubleshooting.md` and the openstack `README.md` to use the full, proper name of RHCOS.

/cc @mrguitar @jlebon @miabbott @darkmuggle 

Related: https://github.com/openshift/installer/pull/1359

### CODE REVIEW

File: docs/user/openstack/README.md [hunk 1]
```diff
@@ -45,7 +45,7 @@ enough to store the ignition config files, so they are served by swift instead.
   values. For example (as an OpenStack admin) `openstack quota set --secgroups 100 --secgroup-rules 1000 <project>`
 
 * The installer requires a proper RHCOS image in the OpenStack cluster or project:
-`openstack image create --container-format=bare --disk-format=qcow2 --file redhat-coreos-${RHCOSVERSION}-openstack.qcow2 redhat-coreos-${RHCOSVERSION}`
+`openstack image create --container-format=bare --disk-format=qcow2 --file redhat-enterprise-linux-coreos-${RHCOSVERSION}-openstack.qcow2 redhat-enterprise-linux-coreos-${RHCOSVERSION}`
```
Comment (wking) [inline docs/user/openstack/README.md]: Do we want to move to `red-hat` while we're at it?

Comment (wking) [reply docs/user/openstack/README.md]: Or use `rhcos-${RHCOSVERSION}`?

Comment (ashcrow) [reply docs/user/openstack/README.md]: /cc @mrguitar :arrow_up:

Comment (mrguitar) [reply docs/user/openstack/README.md]: I think something akin to RHCOS-${version}-${platform}.type be easier. RHEL follows a similar pattern today: rhel-server-7.5-x86_64-dvd.iso. Expanding red-hat-enterprise-linux seems cumbersome to me.

Comment (ashcrow) [reply docs/user/openstack/README.md]: updating

**Comment by ashcrow** [comment]
Updated per review. PTAL.

File: docs/user/troubleshooting.md [hunk 2]
```diff
@@ -79,13 +79,13 @@ The most important thing to look at on the bootstrap node is `bootkube.service`.
 
 ### etcd Is Not Running
 
-etcd is started and managed by the Kubelet as a static pod. This requires a newer Kubelet which started shipping with version 47.29 of Red Hat CoreOS. The OS version can be checked using the following command:
+etcd is started and managed by the Kubelet as a static pod. This requires a newer Kubelet which started shipping with version 47.29 of Red Hat Enterprise Linux CoreOS. The OS version can be checked using the following command:
```
Comment (wking) [inline docs/user/troubleshooting.md]: Should we drop this paragraph through the next?  I don't expect anyone is running RHCOS <27.29 anymore.  We probably want to keep "During the bootstrap process, the Kubelet may emit errors like..." and later.

Comment (ashcrow) [reply docs/user/troubleshooting.md]: Sure. Updating.

Comment (ashcrow) [reply docs/user/troubleshooting.md]: Removed.

**Pull request merged**